### DISCUSSION TITLE
Custom link on left sidebar to access backend

### DISCUSSION THREAD

**Discussion by Zmove** [webstudio-is/webstudio-community#170] [Ideas]
Hello,

It would be great to have an option in the project setting to add a link to the CMS backend for easier access.

Like that.
![image](https://github.com/user-attachments/assets/e7b9e035-13a3-42e1-b4f8-846d86f5a1de)

And we could manage that link somewhere here, to put a link to hygraph, airtable, custom backend, whatever...
![image](https://github.com/user-attachments/assets/73b36ad6-436a-4ed1-99fd-1f6ef347dd87)

Eventually being able to choose the icon, but if it's a generic icon, it's OK tier. 

Should be quite simple to add, and improve the integration feeling between the frontend (webstudio) and the backend (the cms)

**Comment by kof** [comment]
Basically you want to be able to add a dock icon with any random link to the builder.

**Reply by kof** [reply]
What we already planed at some point is having apps installable in the dock. Once they are installed they open a panel with their settings which could have anything you want including a link/button to a CMS.

**Reply by Zmove** [reply]
Sounds perfect to have that feature with an app
